src/Stats.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <QLineEdit>
+#include <QMap>
+#define UNSOLVED_SCORE 2000
```

Comment:
Nitpick: this is only used once, so you might as well just hardcode `2000` down below. If anything, it prevents accidental collisions of preprocessor macros.